pkg/utils/kubelessutil.go

```diff
@@ -141,7 +141,7 @@ func getProvisionContainer(function, checksum, fileName, handler, contentType, r
 
 	// Copy deps file to the installation path
 	runtimeInf, err := lr.GetRuntimeInfo(runtime)
-	if err == nil && runtimeInf.DepName != "" {
```

Comment:
as I said in the issue, I would just add the `deps` inclusion to the type, so I would do `strings.Contains(contentType, "deps")`